### PR TITLE
prepared releasenotes for 2.17.1

### PR DESCRIPTION
**Pull request by cdietrich**
prepared releasenotes for 2.17.1

Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
DO NOT MERGE YET

File: xtend-website/_posts/releasenotes/2019-04-02-version-2-17-1.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+date:   2019-04-02
+categories: releasenotes
+published: true
+---
+
+This is a hotfix releases. Here is the list of fixed issues:
+
+- There is a [Bug](https://bugs.eclipse.org/bugs/show_bug.cgi?id=545343) in the Xtext Builder that leads to non Xtext Projects being built by Xtext.
+- The [LSP4J](https://github.com/eclipse/xtext-core/issues/1059) version in Xtext 2.17.0 does not match the one in Eclipse 2019-03.
+- The [Xtend Batch Compiler](https://github.com/eclipse/xtext-xtend/issues/750) prefers classes found in the Classpath over local classes.
```
Comment (szarnekow) [inline xtend-website/_posts/releasenotes/2019-04-02-version-2-17-1.md]: Classpath -> classpath

File: xtend-website/_posts/releasenotes/2019-04-02-version-2-17-1.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+layout: post
+title:  Xtend 2.17.1 Release Notes
+date:   2019-04-02
+categories: releasenotes
+published: true
+---
+
+This is a hotfix releases. Here is the list of fixed issues:
+
+- There is a [Bug](https://bugs.eclipse.org/bugs/show_bug.cgi?id=545343) in the Xtext Builder that leads to non Xtext Projects being built by Xtext.
```
Comment (szarnekow) [inline xtend-website/_posts/releasenotes/2019-04-02-version-2-17-1.md]: [Bug] -> [bug]

Comment (szarnekow) [reply xtend-website/_posts/releasenotes/2019-04-02-version-2-17-1.md]: Projects -> projects

File: xtext-website/_posts/releasenotes/2019-04-02-version-2-17-1.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+layout: post
+title:  Xtext 2.17.1 Release Notes
+date:   2019-04-02
+categories: releasenotes
+published: true
+---
+
+This is a hotfix releases. Here is the list of fixed issues:
+
+- There is a [Bug](https://bugs.eclipse.org/bugs/show_bug.cgi?id=545343) in the Xtext Builder that leads to non Xtext Projects being built by Xtext.
```
Comment (szarnekow) [inline xtext-website/_posts/releasenotes/2019-04-02-version-2-17-1.md]: See above

File: xtend-website/_posts/releasenotes/2019-04-02-version-2-17-1.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title:  Xtend 2.17.1 Release Notes
+date:   2019-04-02
+categories: releasenotes
+published: true
+---
+
+This is a hotfix releases. Here is the list of fixed issues:
```
Comment (szarnekow) [inline xtend-website/_posts/releasenotes/2019-04-02-version-2-17-1.md]: releases -> release

File: xtext-website/_posts/releasenotes/2019-04-02-version-2-17-1.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title:  Xtext 2.17.1 Release Notes
+date:   2019-04-02
+categories: releasenotes
+published: true
+---
+
+This is a hotfix releases. Here is the list of fixed issues:
```
Comment (szarnekow) [inline xtext-website/_posts/releasenotes/2019-04-02-version-2-17-1.md]: See above

File: xtend-website/_posts/releasenotes/2019-04-02-version-2-17-1.md (hunk 4 above)
Comment (cdietrich) [reply xtend-website/_posts/releasenotes/2019-04-02-version-2-17-1.md]: fixed

File: xtend-website/_posts/releasenotes/2019-04-03-version-2-17-1.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+title:  Xtend 2.17.1 Release Notes
+date:   2019-04-03
+categories: releasenotes
+published: true
+---
+
+This is a hotfix release. Here is the list of fixed issues:
+
+- There is a [bug](https://bugs.eclipse.org/bugs/show_bug.cgi?id=545343) in the Xtext Builder that leads to non Xtext projects being built by Xtext.
+- The [LSP4J](https://github.com/eclipse/xtext-core/issues/1059) version in Xtext 2.17.0 does not match the one in Eclipse 2019-03.
```
Comment (kthoms) [inline xtend-website/_posts/releasenotes/2019-04-03-version-2-17-1.md]: "did not match"; which versions?

**Pull request merged**